pulpanalytics/models.py

```diff
@@ -31,6 +31,14 @@ def __str__(self):
         return f"SystemID={self.system.system_id}, Name={self.name}, Version={self.version}"
 
 
+class PostgresqlVersion(models.Model):
+    version = models.TextField()
```

Comment:
Can we just add a `postgres_version` field to the `System` model? It would nicely reflect the representation in the `Telemetry` object as well.

Reply:
Next push is coming with this. +1